### PR TITLE
Carbon logging fix

### PR DESCRIPTION
**Pull request by JasonIamAUnixAdmin**

### CODE REVIEW

File: monitor/etc/sensu/conf.d/check_system.json [hunk 1]
```diff
@@ -25,6 +25,12 @@
       "command": "/etc/sensu/plugins/vmstat-metrics.rb --scheme sensu.:::name:::",
       "interval": 60,
       "subscribers": [ "all" ]
+    },
+    "fs_checks": {
+      "handlers": ["mailer-ses"],
+      "command": "/opt/sensu/embedded/bin/check-disk-usage.rb -w 85 -c 90",
```
Comment (cpenner461) [inline monitor/etc/sensu/conf.d/check_system.json]: We were going to change this to warn at 75 and critical at 85 ?

File: monitor/etc/sensu/conf.d/check_system.json [hunk 2]
```diff
@@ -25,6 +25,12 @@
...
+      "interval": 60,
+      "subscribers": ["sysmon"]
```
Comment (cpenner461) [inline monitor/etc/sensu/conf.d/check_system.json]: Change this to `all` ?

**Comment by JasonIamAUnixAdmin** [comment]
You are correct. changes made

**Comment by JasonIamAUnixAdmin** [comment]
Closing this PR. Killing all branches and re-creating changes to fix this... just easier that way

**Pull request closed without merging**